charts/k3s/values.yaml

```diff
@@ -194,6 +194,10 @@ service:
   # Configuration for LoadBalancer service type
   externalIPs: []
   externalTrafficPolicy: ""
+  # Extra Labels for the service
+  labels: {}
+  # Extra Annotations for the service
+  annotations: {}
```

Comment:
besides adding this to the k3s chart would be great if you could add this to the k0s and k8s chart as well